antora/docs/modules/ROOT/pages/authoring.adoc

```diff
@@ -35,5 +35,16 @@ rule.
   `custom.rule_data.allowed_registry_prefixes`, allowing a single source of truth for policy rule
   evaluation and documentation. For best results, each key in the `custom.rule_data` object
   should be a noun.
+* `custom.collection`: A list of strings representing a list of rule collections
+  that the policy rule is included in.
 
 The annotations must be defined at the `rule` https://www.openpolicyagent.org/docs/latest/annotations/#scope[scope].
+
+== Package annotations
+
+The use of package annotations is constrained to documenting the rule
```

Comment:
We do actually use package annotations to add some blurb under the package heading. So this might need a revision.

    Package annotations can be used to give a title and description to a package. They can also
    be used in an empty package to add a title and description to a collection.

Something like this...

Reply:
`Use the package name "policy.<kind>.collection.<collectionName>" in an otherwise empty package for collection annotations.`